site/assets/adopters/daocloud.yaml

```
@@ -0,0 +1,11 @@
+name: DaoCloud, Inc.
+description: Testing DaoCloud Enterprise(Private Cloud Platform)
```

```suggestion
description: DaoCloud uses KWOK to test their kubernetes-based cloud platform, namely DaoCloud Enterprise
```
Could we provide a complete desc here?